SharedContent/cpp-winui-template/WinUI3TemplateCpp/Scenario1_ShortName.xaml.cpp

```diff
@@ -0,0 +1,46 @@
+﻿#include "pch.h"
+#include "Scenario1_ShortName.xaml.h"
+#if __has_include("Scenario1_ShortName.g.cpp")
+#include "Scenario1_ShortName.g.cpp"
+#endif
+
+using namespace winrt;
+using namespace Microsoft::UI::Xaml;
+using namespace Microsoft::UI::Xaml::Controls;
+
+
+namespace winrt::WinUI3TemplateCpp::implementation
+{
+
+    WinUI3TemplateCpp::MainPage Scenario1_ShortName::rootPage{ nullptr };
+
+    Scenario1_ShortName::Scenario1_ShortName()
+    {
+        InitializeComponent();
+        Scenario1_ShortName::rootPage = WinUI3TemplateCpp::MainPage::Current();
+    }
+
+    
+    void Scenario1_ShortName::SuccessMessage_Click(winrt::Windows::Foundation::IInspectable const& sender, winrt::Microsoft::UI::Xaml::RoutedEventArgs const& e)
```

Comment:
Consider removing the parameter names, or you're going to get VC complaining about "unreferenced formal parameter". So say this:

```c++
void Scenario1_ShortName::SuccessMessage_Click(winrt::Windows::Foundation::IInspectable const&, RoutedEventArgs const&)
```

Reply:
removed all parameter names that cause "unreferenced formal parameter" warnings